modules/channel-web/src/messaging-migration/down-pg.ts

```diff
@@ -30,11 +30,14 @@ export class MessagingPostgresDownMigrator extends MessagingDownMigrator {
   protected async cleanup() {
     await super.cleanup()
 
-    await this.trx.raw('DROP TABLE msg_messages CASCADE')
-    await this.trx.raw('DROP TABLE msg_conversations CASCADE')
-    await this.trx.raw('DROP TABLE msg_users CASCADE')
-    await this.trx.raw('DROP TABLE msg_clients CASCADE')
-    await this.trx.raw('DROP TABLE msg_providers CASCADE')
+    const tables = await this.trx('pg_catalog.pg_tables')
+      .select('tablename')
+      .andWhere('tablename', 'like', 'msg_%')
+
+    for (const table of tables) {
+      await this.trx.raw(`DROP TABLE ${table.tablename} CASCADE`)
+    }
+
     await this.trx.raw('DROP EXTENSION pgcrypto;')
```

Comment:
```suggestion
    await this.trx.raw('DROP EXTENSION IF EXISTS pgcrypto;')
```

This way, even if you add `MIG_MESSAGING_FAST=true` only when running down migrations, I won't fail. And maybe we should do the same in the cleanup function of the up-pg mig, just to be extra safe?

Reply:
We have a `CREATE EXTENSION IF NOT EXISTS pgcrypto;` at the start of this down migration so this is not necessary.